pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -689,6 +690,67 @@ func TestJavaInjection(t *testing.T) {
 				os.Remove(testfile)
 			},
 		},
+		{
+			// Test the java jdk client https request is working
+			name: "java_jdk_client_httpbin",
+			context: testContext{
+				extras: make(map[string]interface{}),
+			},
+			preTracerSetup: func(t *testing.T, ctx testContext) {
+				o, err := exec.Command("java", "-version").CombinedOutput()
+				if err != nil {
+					t.Fatalf("java -version failed to exec %v", err)
+				}
+				t.Skipf("unsure how to parse different java -version %s", string(o))
```

Comment:
this is always skipped? Is that intentional?

Reply:
yes it is intentional as we don't support the java version > 15 currently
openjdk don't have a stable java -version output (like 1.8.0 vs 17.0.32) java8 vs java17
So I put this log for the moment to collect versions, host java attach is less complicated than container, may I will drop it after anyway

Reply:
I dropped the test as I have logs now